minimal.js

```diff
@@ -208,34 +221,48 @@ module.exports = function (dirname, keys, opts) {
     else flush.push(cb)
   }
 
+  db.addBoxer = function (boxer) {
+    boxers.push(boxer)
+  }
+
   db.addUnboxer = function (unboxer) {
     unboxers.push(unboxer)
   }
 
-  db.unbox = function (data, key) {
-    return unbox(data, unboxers, key)
-  }
-  db.box = function (opts, state) {
-    var content = opts.content
-    var recps = opts.content.recps
-    if (recps) {
-      const isNonEmptyArrayOfFeeds = isArray(recps) && recps.every(isFeed) && recps.length > 0
-      if (isFeed(recps) || isNonEmptyArrayOfFeeds) {
-        recps = opts.content.recps = [].concat(recps) // force to array
-        return box(opts.content, recps)
-      } else {
-        throw new Error(
-          'private message recipients must be valid feeds, '+
-          'was:' + JSON.stringify(recps)
-        )
-      }
+  /* TODO extract to ssb-private */
+  var box1 = {
+    box: function (content, recps) {
+      if (!recps.every(isFeed)) return null
```

Comment:
IIRC Patchbay sets `recps` as `{ link, name }`, is that something we want to support here?

Reply:
I don't want to support everyone and their dog. I just want it to be an Array of `feedIds`. Keep in mind this is just the boxer, it doesn't change anything already published.

Also, I'm _fairly_ sure my code doesn't change any behaviour here...

Reply:
Hey patchbay stuff was written before the magical `recps` auto-encryption (I think). Here's the code in patchcore which is doing the encryption for patchbay @christianbundy : https://github.com/ssbc/patchcore/blob/master/sbot.js#L163-L168

TL;DR - this isn't a breaking change

Reply:
Sweet! Thanks for learning me some history, I don't remember `recps` auto-encryption even being a new thing, but that's great info to have.